src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_module_constants.h

```diff
@@ -145,6 +145,36 @@ const char kTP[] = "TP";
 const char kREG[] = "REG";
 // RdsData struct
 
+// SisData struct
+const char kStationShortName[] = "stationShortName";
+const char kStationIDNumber[] = "stationIDNumber";
+const char kStationLongName[] = "stationLongName";
+const char kStationLocation[] = "stationLocation";
+const char kStationMessage[] = "stationMessage";
+// SisData struct
+
+// GPSData struct
+const char kLongitudeDegrees[] = "longitudeDegrees";
+const char kLatitudeDegrees[] = "latitudeDegrees";
+const char kUtcYear[] = "utcYear";
+const char kUtcMonth[] = "utcMonth";
```

Comment:
@mkorniichuk  where do we use all this constants?

Reply:
@LuxoftAKutsan For now, some of them are used for unit testing.

Reply:
@mkorniichuk please keep only constants that are used ( only in UT is OK)